http/headers/set-cookie.json

```diff
@@ -406,6 +413,96 @@
               }
             }
           },
+          "schemeful": {
+            "__compat": {
+              "description": "Schemeful",
```

Comment:
@ddbeck Since this is already a subfeature of `SameSite`, maybe it's not necessary to state it again; same approach as `secure_context_required`. 

But I do agree some more information is needed, so how about this?
 
```suggestion
              "description": "URL scheme-aware (\"schemeful\")",
```

Reply:
Good point about restating the parent. I really like this version; I'm going to commit the suggestion and merge the PR.